Content.Server/Power/EntitySystems/PowerNetSystem.cs

```diff
@@ -297,25 +305,13 @@ private void ReconnectNetworks()
 
         private void UpdateApcPowerReceiver()
         {
-            var appearanceQuery = GetEntityQuery<AppearanceComponent>();
-            var metaQuery = GetEntityQuery<MetaDataComponent>();
-            var enumerator = AllEntityQuery<ApcPowerReceiverComponent>();
-            while (enumerator.MoveNext(out var uid, out var apcReceiver))
+            var enumerator = AllEntityQuery<ApcPowerReceiverComponent, MetaDataComponent>();
+            while (enumerator.MoveNext(out var uid, out var apcReceiver, out var metaDataComponent))
             {
-                var powered = apcReceiver.Powered;
-                if (powered == apcReceiver.PoweredLastUpdate)
+                if (metaDataComponent.EntityPaused)
```

Comment:
IIRC EntityQueryEnumerator returns all the non-paused entities so if you use that this check isnt required

Reply:
This uses AllEntityQuery which does not have any checks